scripts/profiler/rtt_nordic_profiler_host.py

```diff
@@ -190,12 +215,25 @@ def _read_single_event_rtt(self):
             signum = False
             if i[0] == 's':
                 signum = True
-            buf = self._read_bytes(self.config['rtt_data_channel'], 4)
+            buf = self._read_bytes(4)
             data.append(int.from_bytes(buf, byteorder=self.config['byteorder'],
                                        signed=signum))
         return Event(id, timestamp, data)
 
+    def _read_remaining_events(self):
+        self.reading_data = False
+        while self.bcnt != 0:
```

Comment:
what if buffer stores only part of the event?

Reply:
I think it is not possible when everything works normally - microcontroller writes data as whole events and we are making additional reads when there is a lot of data in the buffer.